### ISSUE TITLE
ReduceLROnPlateau` scheduler 

### ISSUE THREAD

**Issue by MrWD10** [LearnedVector/A-Hackers-AI-Voice-Assistant#29]
/usr/local/lib/python3.6/dist-packages/pytorch_lightning/trainer/optimizers.py in configure_schedulers(self, schedulers, monitor)
    113                 if monitor is None:
    114                     raise MisconfigurationException(
--> 115                         '`configure_optimizers` must include a monitor when a `ReduceLROnPlateau` scheduler is used.'
    116                         ' For example: {"optimizer": optimizer, "lr_scheduler": scheduler, "monitor": "metric_to_track"}'
    117                     )

MisconfigurationException: `configure_optimizers` must include a monitor when a `ReduceLROnPlateau` scheduler is used. For example: {"optimizer": optimizer, "lr_scheduler": scheduler, "monitor": "metric_to_track"}
any suggestions?

**Comment by Jochen-sys** [comment]
@MrWD10 Did you fix this error and when yes, how?
